Dockerfile.rocm.ubi

```diff
@@ -178,7 +197,13 @@ RUN --mount=type=bind,from=build_amdsmi,src=/install,target=/install/amdsmi/ \
 RUN umask 002 && \
     useradd --uid 2000 --gid 0 vllm && \
     mkdir -p /licenses && \
-    chmod g+rwx $HOME /usr/src /workspace
+    chmod g+rwx $HOME /usr/src /workspace && \
+    chmod 0664 /var/log/rocm_smi_lib
+
+# Install libsodium for Tensorizer encryption
+RUN --mount=type=bind,from=libsodium-builder,src=/usr/src/libsodium,target=/usr/src/libsodium \
+    cd /usr/src/libsodium \
+    && make install
```

Comment:
How about to move this before the `uv pip install` step above?

Reply:
sure